src/components/DuosHeader.js

```diff
@@ -230,6 +230,29 @@ class DuosHeader extends Component {
       isSigningOfficial = currentUser.isSigningOfficial;
     }
 
+    const hasTwoOrMoreRoles = () => {
+      let roles = 0;
+      if (isAdmin) {
+        roles += 1;
+      }
+      if (isResearcher) {
+        roles += 1;
+      }
+      if (isMember) {
+        roles += 1;
+      }
+      if (isChairPerson) {
+        roles += 1;
+      }
+      if (isDataOwner) {
+        roles += 1;
+      }
+      if (isSigningOfficial) {
+        roles += 1;
+      }
+      return roles >= 2;
+    };
```

Comment:
This can definitely be a lot simpler. The `currentUser` has a list of roles on it ... we just need to count the distinct rolenames from that list.

Reply:
Here's how to get all of the distinct role names ... 

```
import { isNil, uniq, map } from 'lodash/fp';
...
      const roleNames = uniq(map("name")(currentUser.roles));
      console.log(JSON.stringify(roleNames));
```
That gives my superadmin user the following on dev:
```
["Alumni","Admin","Researcher","DataOwner","Chairperson","SigningOfficial"]
```